tests/queries/0_stateless/02845_gcd_codec.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE IF EXISTS aboba;
```

Comment:
let's add a stronger test. smth like
* create two tables: one with codec and one without
* fill with the same random data (e.g. generate by this function: https://clickhouse.com/docs/en/sql-reference/table-functions/generate)
* check that both return the same content. e.g. some checksums over the columns match